src/components/HdIcon.vue

```diff
@@ -53,6 +67,27 @@ export default {
 
       return svg;
     },
+    addAccessibilityTags(svg) {
+      const randId = Math.floor(Math.random() * 1000);
+      const attrs = [];
+      if (this.title) {
+        const titleId = `${this.id || randId}-title`;
+        const titleTag = document.createElementNS('http://www.w3.org/2000/svg', 'title');
+        titleTag.innerHTML = this.title;
+        titleTag.id = titleId;
+        svg.insertBefore(titleTag, svg.firstChild);
+        attrs.push(titleId);
+      }
+      if (this.description) {
+        const descId = `${this.id || randId}-desc`;
+        const descTag = document.createElementNS('http://www.w3.org/2000/svg', 'desc');
+        descTag.id = descId;
+        svg.appendChild(descTag);
+        attrs.push(descId);
+      }
+      if (attrs.length) svg.setAttribute('aria-labelledby', attrs.join(' '));
```

Comment:
Here you're using `aria-labelledby` for both, the title and the description, which is (weirdly) recommended in the article, but I think [aria-describedby](https://www.w3.org/TR/wai-aria/#aria-describedby) will be better for the description, we can keep using `aria-labelledby` for the title though.